share/completions/mtr.fish

```diff
@@ -0,0 +1,41 @@
+complete -c mtr -x -a "(__fish_print_hostnames)"
+
+complete -c mtr -s h -l help -d 'display this help and exit'
```

Comment:
We generally capitalize the descriptions, and this isn't "this help", so it would just be

```
Display help and exit
```